### DISCUSSION TITLE
İbax İzle Kazan

### DISCUSSION THREAD

**Discussion by Majezikonline** [IBAX-io/go-ibax#498] [Ideas]
### Clear and concise description of the problem

İbax üzerinden projeye ait bilgilendirici videolar eklenerek, videoyu izleyen herkese belli bir miktarda ibax verilmesi.

### Suggested solution

Siteye veya mobil uygulamaya bilgilendirici, açıklayıcı ve geliştirici videolar eklenmesi.  ibax için katkı sağlayabilecek videolar eklendikten sonra insanların fikirlerinin ibax'a sunulması 

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by Akrepkapo** [comment]
Bence de haklı olabilir

**Comment by canrasim** [comment]
bu tarz bir etkinlik iyi olabilir ancak bilgilerin daha kapsayıcı alınması lazım
